packages/openspp/src/Adaptor.js

```diff
@@ -57,27 +57,43 @@ async function login(state) {
   return state;
 }
 
+/**
+ * resolve input domain
```

Comment:
I appreciate the documentation on this but just for future reference, you don't need to add JSDocs to internal functions like this :) You only need to JSDoc the exported public functions - the stuff users will use in job code.

Don't remove this, it's great, just bear it in mind for next time :)